ydb/core/tx/datashard/build_index.cpp

```diff
@@ -478,7 +478,8 @@ class TBuildScanUpload: public TActor<TBuildScanUpload<Activity>>, public NTable
             progress->Record.SetRequestSeqNoGeneration(SeqNo.Generation);
             progress->Record.SetRequestSeqNoRound(SeqNo.Round);
 
-            progress->Record.SetLastKeyAck(TSerializedCellVec::Serialize(LastUploadedKey.GetCells()));
+            // TODO(mbkkt) ReleaseBuffer isn't possible, we use LastUploadedKey for logging
+            progress->Record.SetLastKeyAck(LastUploadedKey.GetBuffer());
```

Comment:
Doesn't look like refactoring.
Change of data here, removal of sending responses below.
Explanation is needed.

Reply:
There is no change of data here, it's same data.
Lets me explain:
LastUploadedKey -- already TSerializedCellVec (buffer + vector of cells)
Before by changes we for some reason instead of just copy buffer construct same buffer from cells.

So there is no behaviour change